workflows/rnavar.nf

```diff
@@ -49,9 +49,6 @@ WorkflowRnavar.initialise(params, log)
 
 for (param in checkPathParamList) if (param) file(param, checkIfExists: true)
 
-if (params.input) { ch_input = file(params.input) } else { exit 1, 'Input samplesheet not specified!' }
-if (!params.star_index && !params.gtf && !params.gff){ exit 1, "GTF|GFF3 file is required to build a STAR reference index! Use option --gtf|--gff to provide a GTF|GFF file." }
```

Comment:
what about this warning?

Reply:
nf-validation does it

Reply:
tried with `--star_index=false --gtf=false --gff=false` and got an error that was not there, so fairly sure this is dealt with somewhere else